internal/config/testdata/telegraf-agent.toml

```diff
@@ -190,6 +190,12 @@
   timeout = 100
   # A list of Mesos masters, default value is localhost:5050.
   masters = ["localhost:5050"]
+  # Default protocol for communication. Use https if you have authentication!
+  protocol = "http"
```

Comment:
I would prefer if the protocol scheme were prepended to the server, like `masters = ["http://localhost:5050"]`. You can assume `http` if there is no protocol